lambdatimes/src/components/Content/Content.js

```diff
@@ -18,37 +18,33 @@ export default class Content extends Component {
 
   componentDidMount() {
     // Once the component has mounted, get the data and reflect that data on the state.
+  this.setState({tabs: tabData})
+  this.setState({cards: cardData})
```

Comment:
Here you can put both these into one setState like this, this.setState({ tabs: tabData, cards: cardData)} and get the same result